examples/start_heartbeat.rs

```diff
@@ -0,0 +1,45 @@
+use std::{
```

Comment:
Missing `std::env` and `dotenv::dotenv` caused the error for `cargo run --example start_heartbeat`. 
The following code needs to be added.   

```rust
use dotenv::dotenv;
use std::env;

#[tokio::main]
async fn main() -> Result<(), Error> {
    dotenv().ok();
    ...
}
```